bindings/node/lib/mongocryptdManager.js

```diff
@@ -7,6 +7,8 @@ const spawn = require('child_process').spawn;
  * An internal class that handles spawning a mongocryptd.
  */
 class MongocryptdManager {
+  static DEFAULT_MONGOCRYPTD_URI = 'mongodb://localhost:27020/?serverSelectionTimeoutMS=1000';
```

Comment:
Why aren't we changing the default to be 127.0.0.1?

Reply:
the default uri is actually spec'd https://mongodb.slack.com/archives/C72LB5RPV/p1670607134515969?thread_ts=1670604198.815689&cid=C72LB5RPV